### DISCUSSION TITLE
Adding New Car

### DISCUSSION THREAD

**Discussion by natrlhy** [teslamate-org/teslamate#2297] [Q&A] [answered]
If I already have a teslamate installation running and, I get another Tesla vehicle, if I'm adding it to the same tesla account as the one teslamate is using, does the new car automatically get added when I take delivery of it? Nothing special I need to do?

Thanks!

**Comment by Dulanic** [comment, marked as answer]
Yes

**Comment by metaben** [comment]
The second car will appear. Nothing you need to do.

**Comment by MarcFinns** [comment]
Not exactly. The car does not appear automatically. I had to restart Teslamate to make our second Tesla appear. Hence I lost the first drives!
